integration/src/test/java/com/tutrit/config/Mappers.java

```diff
@@ -0,0 +1,40 @@
+package com.tutrit.config;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+@Configuration
+public class Mappers {
+
+    @Bean
+    public ObjectMapper objectMapper() {
+        final ObjectMapper om = new ObjectMapper();
+        //Don't fail if additional fields in incoming JSON, just ignore
+        om.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        //Don't fail on incoming JSON missing fields
+        om.configure(DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES, false);
+        om.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
+        //Don't include null fields into json (need for patch methods tests)
+        om.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        om.registerModule(new JavaTimeModule());
+        om.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+        return om;
+    }
+
+    @Bean
+    public MockMvc mockMvc() {
+        return MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
+    }
+
+    @Autowired
+    WebApplicationContext webApplicationContext;
+}
```

Comment:
please, add new line in the end of the file

Reply:
I did